src/main/kotlin/com/yapp/weekand/domain/auth/service/AuthService.kt

```diff
@@ -79,6 +84,50 @@ class AuthService (
 		return true
 	}
 
+	@Transactional
+	fun signUp(signUpInput: SignUpInput): String {
+		if (userRepository.existsUserByEmail(signUpInput.email)) {
+			throw UserEmailDuplicatedException()
+		}
+
+		if (userRepository.existsUserByNickname(signUpInput.nickName)) {
+			throw UserNickNameDuplicatedException()
+		}
+
+		val user = User(nickname = signUpInput.nickName,
+			email = signUpInput.email,
+			marketingAgreed = true,
+			password = passwordEncoder.encode(signUpInput.password))
+		userRepository.save(user)
```

Comment:
```suggestion
		val user = User(
		    nickname = signUpInput.nickName,
			email = signUpInput.email,
			marketingAgreed = true,
			password = passwordEncoder.encode(signUpInput.password)
		)
		
		userRepository.save(user)
```
이런식으로 작성하면 가독성이 좋아질 거 같습니다~

Reply:
오호 알겠씁니다 감사합니당 👍🏻